integrations/jenkins/.port/resources/blueprints.json

```diff
@@ -17,13 +17,15 @@
                         "created",
                         "unknown",
                         "passing",
-                        "failing"
+                        "failing",
+                        "aborted"
                     ],
                     "enumColors": {
                         "passing": "green",
                         "created": "darkGray",
                         "failing": "red",
-                        "unknown": "orange"
+                        "unknown": "orange",
+                        "aborted": "gray"
```

Comment:
@jortegac, thank you for these changes. Just one comment: gray is not a valid colour option; use `lightGray` instead

Reply:
@jortegac once fixed, please merge the main branch into your fork to resolve all the conflicts and then we can merge.